Cargo.lock

```diff
@@ -1939,6 +1939,12 @@ dependencies = [
  "serde_json",
 ]
 
+[[package]]
+name = "jzon"
+version = "0.12.5"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "17ab85f84ca42c5ec520e6f3c9966ba1fd62909ce260f8837e248857d2560509"
```

Comment:
Question: is there a particular reason why we're using `jzon` or a new package, instead of something like `serde_json`? (I have not used `jzon` before)

Reply:
+1. Let's just use `serde_json::Value` for this to avoid adding a new dependency.

Reply:
Ah indeed. Let's go with `serde_json` then!